### ISSUE TITLE
Alert component: props updates

### ISSUE THREAD

**Issue by robinpiets** [Amsterdam/amsterdam-styled-components#1245]
As discussed with @jonkoops and @chrisvanmook we're going to update the props for the Alert component.

### level
`level` will hold the information about the color:

```js 
type Level = 'neutral' | 'error' | 'info' | 'warning'

type Props = {
   level?: Level // default: 'neutral'
   ...
```

`level` can have these values:
- neutral is default and will be grey ![#E6E6E6](https://via.placeholder.com/15/E6E6E6/000000?text=+)
- info will be blue ![#004699](https://via.placeholder.com/15/004699/000000?text=+)
- warning will be yellow ![#FEC813](https://via.placeholder.com/15/FEC813/000000?text=+)
- error will be red ![#EC0000](https://via.placeholder.com/15/EC0000/000000?text=+)

### outline

`outline` will decide if the color of the level should be on the background (`false`) or the border / box-shadow (`true`)

```js 
outline: boolean // default: false`
```

**Comment by robinpiets** [comment]
Fixed in #1295
